src/main/resources/view/PersonListCard.fxml

```diff
@@ -8,29 +8,35 @@
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.image.Image?>
```

Comment:
Image seems to be an unused import? It might be good to remove it.

Reply:
Used in line 72. ImageView provides the UI with the input image, which is read using the Image class.